test-requirements.txt

```diff
@@ -13,3 +13,4 @@ sqlalchemy
 sqlalchemy_utils
 mypy
 sqlalchemy-stubs
+psutil
```

Comment:
This is a requirement of HTEX, not of the test suite, I think? (process-worker-pool wants to import psutil...)

Reply:
I've fixed this now.